docs/tutorials/elasticsearch.ipynb

```diff
@@ -0,0 +1,750 @@
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "LI0Mmp_dT7yu"
+      },
+      "source": [
+        "Choose a subset of features."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "M0X9LEKoUfbU"
+      },
+      "outputs": [],
+      "source": [
+        "all_inputs = []\n",
+        "encoded_features = []\n",
+        "\n",
+        "# Numeric features.\n",
+        "for header in ['PhotoAmt', 'Fee']:\n",
+        "  numeric_col = tf.keras.Input(shape=(1,), name=header)\n",
+        "  normalization_layer = get_normalization_layer(header, train_ds)\n",
+        "  encoded_numeric_col = normalization_layer(numeric_col)\n",
+        "  all_inputs.append(numeric_col)\n",
+        "  encoded_features.append(encoded_numeric_col)\n",
+        "\n",
+        "# Categorical features encoded as string.\n",
+        "categorical_cols = ['Type', 'Color1', 'Color2', 'Gender', 'MaturitySize',\n",
+        "                    'FurLength', 'Vaccinated', 'Sterilized', 'Health', 'Breed1']\n",
+        "for header in categorical_cols:\n",
+        "  categorical_col = tf.keras.Input(shape=(1,), name=header, dtype='string')\n",
+        "  encoding_layer = get_category_encoding_layer(header, train_ds, dtype='string',\n",
+        "                                               max_tokens=5)\n",
+        "  encoded_categorical_col = encoding_layer(categorical_col)\n",
+        "  all_inputs.append(categorical_col)\n",
+        "  encoded_features.append(encoded_categorical_col)  "
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "x84lZJY164RI"
+      },
+      "source": [
+        "## Build, compile and train the model\n"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "uuHtpAMqLqmv"
+      },
+      "outputs": [],
+      "source": [
+        "# Set the parameters\n",
+        "\n",
+        "OPTIMIZER=\"adam\"\n",
+        "LOSS=tf.keras.losses.BinaryCrossentropy(from_logits=True)\n",
+        "METRICS=['accuracy']\n",
+        "EPOCHS=10\n"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "7lBmxxuj63jZ"
+      },
+      "outputs": [],
+      "source": [
+        "# Convert the feature columns into a tf.keras layer\n",
+        "all_features = tf.keras.layers.concatenate(encoded_features)\n",
+        "\n",
+        "# design/build the model\n",
+        "x = tf.keras.layers.Dense(32, activation=\"relu\")(all_features)\n",
+        "x = tf.keras.layers.Dropout(0.5)(x)\n",
+        "x = tf.keras.layers.Dense(64, activation=\"relu\")(x)\n",
+        "x = tf.keras.layers.Dropout(0.5)(x)\n",
+        "output = tf.keras.layers.Dense(1)(x)\n",
+        "model = tf.keras.Model(all_inputs, output)\n",
+        "\n",
+        "print(model.summary())"
```

Comment:
If the model isn't a simple sequential, you can't see the structure with summary. prefer `plot_model`:

```
tf.keras.utils.plot_model(model, rankdir='LR', show_shapes=True)
```

Reply:
addressed in the commit